dev/src/codewind/project/Project.ts

```diff
@@ -464,6 +466,77 @@ export default class Project implements vscode.QuickPickItem {
         });
     }
 
+    public async onLoadRunnerUpdate(event: {projectID: string, status: string, timestamp: string}): Promise<void> {
+        Log.d(`${this.name} load runner status changed to ${event.status}`, event);
+        switch (event.status) {
+            case "hcdReady":
+            case "profilingReady":
+                Log.d("Profiling data is ready to be saved to workspace");
+                const loadTestPath = path.join(this.localPath.fsPath, "load-test");
+                const timestampPath = path.join(loadTestPath, event.timestamp);
+                let fileName = "";
+                if (this.language === "java") {
+                    fileName = "profiling.hcd";
+                } else if (this.language === "nodejs") {
+                    fileName = "profiling.json";
+                }
+                const profilingOutPath = path.join(timestampPath, fileName);
+                try {
+                    await fs.promises.mkdir(timestampPath, {recursive: true});
+                    const fd = await fs.promises.open(profilingOutPath, "w");
+                    await fd.close();
+                } catch (error) {
+                    Log.e(error);
```

Comment:
Since the first argument to `Log.e` is a `string`, this will print `[Object object]`. Put the `error` as the second arg.

Reply:
`Log.e("", error)` ?

Reply:
instead of "" something like `error writing profiling data to ${profilingOutPath}`